AdobeStockImageAdminUi/view/adminhtml/web/js/components/grid/column/preview/actions.js

```diff
@@ -137,20 +137,47 @@ define([
             this.preview().getAdobeModal().trigger('closeModal');
 
             if (!this.isMediaBrowser()) {
-                this.selectImageInNewMediaGalleryBySearch(this.preview().displayedRecord().title);
+                this.selectImageInNewMediaGalleryBySearch(this.preview().displayedRecord().id);
             } else {
                 this.selectDisplayedImageForOldMediaGallery(this.preview().displayedRecord().path);
             }
         },
 
+        /**
+         * Return adobe stock asset by adobe id
+         *
+         * @param {String} adobeId
+         */
+        getAssetDetails: function (adobeId) {
+            return $.ajax({
+                url: this.getAssetDetailsUrl,
```

Comment:
```suggestion
                url: this.getMediaGalleryAsset,
```
Also please add the filed to defaults